### ISSUE TITLE
Hever: Drop-down menus not visible when a light primary color is selected

### ISSUE THREAD

**Issue by poeticabdul** [Automattic/themes#5271]
## Steps to replicate

Seems that when choosing a light primary color the hover background color for sub menu items is set to white and the text is then no longer visible.

1- Make a drop-down menu
2- Choose a light primary color for the theme. For example `#ffb302`

![Screenshot at Jan 05 10-33-54](https://user-images.githubusercontent.com/4719037/148203314-49870198-5413-4f06-8580-1d6bc85e8554.png)

## Result

Notice the submenu items are not visible

![Screenshot 2022-01-05 at 10 15 26 AM](https://user-images.githubusercontent.com/4719037/148202872-df76a322-a6fa-4c5d-9366-70e5ba8f6043.png)

## Expected

The sub-menu items should be visible so the user can know which one they are clicking on

* User report: 4659869-zen

**Comment by Robertght** [comment]
Closing in favor of https://github.com/Automattic/themes/issues/5589
